### PR TITLE
FUTR-129: Automatically generate API routes file based on starter config during init.

### PR DESCRIPTION
**Pull request by abettke**

### CODE REVIEW

File: examples/contact-book/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+services:
+  db:
+    image: postgres:13
+    container_name: 'contact-book-db'
+    ports:
+      - '5432:REPLACE_DB_PORT'
```
Comment (abhiaiyer91) [inline examples/contact-book/docker-compose.yaml:6]: what happened here, the replacements didnt happen?

Comment (abettke) [reply examples/contact-book/docker-compose.yaml:6]: No replacements didn't happen. This is what happens when running inti without these files here.

**Review by calcsam — APPROVED** [review]
LGTM!

File: examples/contact-book/docker-compose.yaml (hunk 1 above)
Comment (adeleke5140) [reply examples/contact-book/docker-compose.yaml:6]: yeah noticed this as well, think it's a bug with a refactored find and replace function

Comment (TheIsrael1) [reply examples/contact-book/docker-compose.yaml:6]: I believe this should be fixed here
https://github.com/kepler-inc/future/pull/166

**Pull request merged**